File: Tareas/Unidad2/a2253332351_practica01/10_AreasPrisma.java
import java.util.Scanner;

public class AreasPrisma {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double perimetro, apotema, altura, aTotal, aLateral, aBase;

           System.out.println("Ingresa el perimetro de la Base: ");
           perimetro = entrada.nextDouble();
           System.out.println("Ingresa la apotema: ");
           apotema = entrada.nextDouble();
           System.out.println("Ingresa la altura: ");
           altura = entrada.nextDouble();

           aBase = (perimetro * apotema) / 2;
           aLateral = (perimetro * altura) / 2;
           aTotal = (2 * aBase) + aLateral;

           System.out.println("El área de la base es: " + aBase);
           System.out.println("El área lateral es: " + aLateral);
           System.out.println("El área total es: " + aTotal);
     }
}

File: Tareas/Unidad2/a2253332351_practica01/10_AreasPrisma.py
# AreasPrisma.py

perimetro = float(input("Ingresa el perimetro de la Base: "))
apotema = float(input("Ingresa la apotema: "))
altura = float(input("Ingresa la altura: "))

aBase = (perimetro * apotema) / 2
aLateral = (perimetro * altura) / 2
aTotal = (2 * aBase) + aLateral

print(f"El área de la base es: {aBase}")
print(f"El área lateral es: {aLateral}")
print(f"El área total es: {aTotal}")
